src/app/admin/products-table/products-table.tsx

```diff
@@ -0,0 +1,32 @@
+import { ScrollArea, Table, TextInput } from '@mantine/core'
+import { Product } from '@prisma/client'
+import { Search } from 'tabler-icons-react'
+import { ProductsTableBody } from './products-table-body'
+import { ProductsTableHeader } from './products-table-header'
+import { useProductTable } from './use-product-table'
+
+interface Props {
+  data: Product[]
+  openEdit: (product: Product) => void
+}
+
+export function ProductsTable({ data, openEdit }: Props) {
+  const { sortedData, sortBy, setSorting, search, handleSearchChange, reverseSortDirection, selection, toggleRow } =
+    useProductTable(data)
+
+  return (
+    <ScrollArea>
+      <TextInput
```

Comment:
Do you think we could/should make the text box highlight red when active for consistency?

Reply:
We can hold off for later. I noticed the text box in the Product form also doesn't have red for active